3rdparty/LICENSE.3rdparty

```diff
@@ -62,7 +60,7 @@ SOFTWARE.
 
 
 ================================================================================
-For EnumClass (hilti/include/3rdparty/enum-class):
+EnumClass
```

Comment:
Should this instead be named `ArticleEnumClass-v2`?

Reply:
ack